src/client/pages/AssessmentHome/FraHome/FraHome.tsx

```diff
@@ -33,7 +33,7 @@ const FraHome: React.FC = () => {
   const displayTabs = sections.length > 1 && Areas.isISOCountry(countryIso)
 
   useEffect(() => {
-    if (Users.getRolesAllowedToEdit({ user, countryIso, cycle }).length > 0) {
+    if (Authorizer.canViewUsers({ countryIso, assessment, cycle, user })) {
```

Comment:
Better to have a common hook in home . useGetUsers to use in fra and pan European home